### DISCUSSION TITLE
Cannot access web interface Yi Home Pro 2K YFUS

### DISCUSSION THREAD

**Discussion by EhV8or** [roleoroleo/yi-hack-Allwinner-v2#804] [Q&A]
Hello! I greatly appreciate you putting all this work together! I have, I think, successfully hacked my Yi Home Pro 2K, but am unable to access the web interface. My ultimate goal is to enable RTMP and add two cameras to my Blue Iris home system.

When I enter the assigned IP address of http://192.168.X.X it does nothing in both Chrome and Edge, and I also cannot access a feed for it on VLC when entered the same way. Per the FAQ, the presence of Factory.done should mean I succesfully hacked it, so I'm clueless as to what my issue is. Also, even though I see in another place to change the name of the configure_wifi_cfg file to use wifi, it does currently connect through the Yi Home app utilizing wifi.

When I put my SD back into my computer I have the following files on the card:

backup
Factory.done/yi-hack/fw_upgrade_in_progress + lower_half_init.ssh
                     /config.sh
                     /configure_wifi.cfg
                     /configure_wifi.sh
                     /factory_test.sh
record
.wtest

I sent you a gratuity as a thank you for providing this hack :) Have a great day!

**Comment by roleoroleo** [comment]
Thank you for your donation.

About your problem, the presence of the file fw_upgrade_in_progress means that the hack doesn't complete the 2nd phase of the process.
I don't know why.

But you should be able to enter with a telnet client and debug the problem.
Please run these commands:
```
ps
ls -laR /tmp/sd
```
